src-docs/src/views/combo_box/combo_box_example.js

```diff
@@ -416,6 +425,34 @@ export const ComboBoxExample = {
       snippet: singleSelectionSnippet,
       demo: <SingleSelection />,
     },
+    {
+      title: 'Single selection with prepended label',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: singleSelectionPrependSource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: singleSelectionHtml,
+        },
+      ],
+      text: (
+        <Fragment>
+          <p>
+            <EuiCode>append</EuiCode> and <EuiCode>prepend</EuiCode> props only
+            work if
+            <EuiCode>singleSelection</EuiCode> prop is not set to{' '}
+            <EuiCode>false</EuiCode> to avoid multi-lines that makes combobox
+            height greater than that of <EuiCode>append</EuiCode> and{' '}
+            <EuiCode>prepend</EuiCode>.
+          </p>
```

Comment:
```suggestion
            The <EuiCode>prepend</EuiCode> and <EuiCode>append</EuiCode> props only
            work if{' '}
            <EuiCode>singleSelection</EuiCode> prop is not set to{' '}
            <EuiCode>false</EuiCode>. This avoids multi-line comboboxes having a height
            greater than that of these elements.
          </p>
```

This sentence is also now duplicative of the `Note` in the prior example. Can you remove that?